src/assets/js/main.js

```diff
@@ -105,15 +111,19 @@ function adjustToc() {
 function createGallery() {
   for (var i = 0; i < arguments.length; i++) {
     const galleryName = arguments[i];
-    const selector = $('#' + galleryName + ' .selector li');
-    
-    selector.hover(function () {
-      switchBanner.call(this, galleryName);
-    });
+    const selectors = document.querySelectorAll('#' + galleryName + ' .selector li')
 
-    selector.focus(function () {
-      switchBanner.call(this, galleryName);
-    });
+    selectors.forEach(selector => {
+      selector.addEventListener('mouseover', function (event) {
+        switchBanner.call(this, galleryName);
+      });
+    })
+    
+    selectors.forEach(selector => {
+      selector.addEventListener('focus', function (event) {
+        switchBanner.call(this, galleryName);
+      });
+    })
```

Comment:
These adjustments look great, but a few suggestions:

- No need to loop over the `selectors` twice, since it's the same collection.
- Remember to end each statement with a semicolon for consistency and to avoid potential issues.
- Consider renaming `event` to just `_` since it is unused.

Potentially like the following:

```suggestion
    selectors.forEach(selector => {
      selector.addEventListener('mouseover', function (_) {
        switchBanner.call(this, galleryName);
      });
      
      selector.addEventListener('focus', function (_) {
        switchBanner.call(this, galleryName);
      });
    });
```